[EDU-6371] Update terminology in devtools docs en (#2)
VitorEltz/azion-docs: src/content/docs/en/pages/devtools/api-graphql/error-responses/error-responses.mdx

---

> title: Custom Pages GraphQL API

Author: ```suggestion
title: Error Responses GraphQL API
```

description: >-
  The GraphQL API returns standard HTTP status codes to indicate an error.
  Whenever you receive a status code while trying to run a request, you'll also

>   receive a Custom Page.

Author: ```suggestion
  receive an Error Responses.
```

> meta_tags: 'graphql, api, azion, query, custom pages'

Author: ```suggestion
meta_tags: 'graphql, api, azion, query, error responses'
```

namespace: documentation_graphql_error_responses
permalink: /documentation/devtools/graphql-api/error-responses/
menu_namespace: graphqlMenu
---

The **GraphQL API** returns standard HTTP status codes to indicate an error. Whenever you receive a status code while trying to run a request, you'll also receive an error message. These errors relate to *authentication*, *limits*, *access*, or query and GraphQL *construction* issues.

> You can receive the following status codes and Custom Pages in a query format:

Author: ```suggestion
You can receive the following status codes and error messages in a query format:
```

description: >-
  The GraphQL API returns standard HTTP status codes to indicate an error.
  Whenever you receive a status code while trying to run a request, you'll also

>   receive an Error Responses.
> meta_tags: 'graphql, api, azion, query, error responses'

Author: ```suggestion
  receive an error message.
meta_tags: 'graphql, api, azion, query, error response'
```

| 404         | The following resource could not be found.                                                                                      | The client_id isn't allowed to access the specific API resource.                     |
| 429         | You have reached the request rate limit!                                                                                        | The request rate limit for the IP address has been exceeded.                         |

> If you receive any of the described Custom Pages, see the rest of the [GraphQL API documentation](/en/documentation/devtools/graphql-api/) to find specifications and correct your request.

Author: ```suggestion
If you receive any of the described error responses, see the rest of the [GraphQL API documentation](/en/documentation/devtools/graphql-api/) to find specifications and correct your request.
```
